templates/newrelic-stack-set-cfct.yaml

```diff
@@ -0,0 +1,326 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: Integrates New Relic with AWS accounts that use AWS Control Tower Customization Framework (CfCT). This deployment launches as a stack set in your AWS Control Tower landing zone.(qs-1rsulpe1p)
+Parameters:
```

Comment:
As discussed, we need to add a parameter for customers to explicitly allow creating cross account access. Default value of that parameter would be "False".

Reply:
explicit param added